packages/services/f-http/src/tests/index.test.js

```diff
@@ -1,11 +1,22 @@
-import HttpClient from '../httpClient';
+import createClient from '../createClient';
 
-describe('f-http', () => {
+describe('createClient', () => {
     beforeEach(() => {});
 
     it('httpClient should be defined', async () => {
         // Arrange & Act
         // Assert
-        expect(HttpClient).toBeDefined();
+        expect(createClient).toBeDefined();
+    });
+
+    it('httpClient should expose expected methods', async () => {
+        // Arrange & Act
+        const httpClient = createClient();
+
+        // Assert
+        expect(httpClient).toBeDefined();
+        expect(httpClient.get).toBeDefined();
+        expect(httpClient.post).toBeDefined();
+        expect(httpClient.readConfiguration).toBeDefined();
```

Comment:
Should there be test cases for null being returned for `data` and for the calls to `errorCallback` throwing?

Reply:
I intend to put some tests in for the GET, PUT, POST, PATCH, DELETE's when they are all in there as part of the same ticket; just trying to keep the PR's manageable and small